src/Libraries/Microsoft.Extensions.AI.Evaluation.Reporting.Azure/Microsoft.Extensions.AI.Evaluation.Reporting.Azure.json

```diff
@@ -37,10 +34,7 @@
       ]
     },
     {
-      // After generating the baseline, manually edit this file to remove primary constructor portion
-      // This is needed until ICSharpCode.Decompiler adds support for primary constructors
-      // See: https://github.com/icsharpcode/ILSpy/issues/829
-      "Type": "sealed class Microsoft.Extensions.AI.Evaluation.Reporting.Storage.AzureStorageResultStore : Microsoft.Extensions.AI.Evaluation.Reporting.IEvaluationResultStore",
+      "Type": "sealed class Microsoft.Extensions.AI.Evaluation.Reporting.Storage.AzureStorageResultStore(Azure.Storage.Files.DataLake.DataLakeDirectoryClient client) : Microsoft.Extensions.AI.Evaluation.Reporting.IEvaluationResultStore",
```

Comment:
As mentioned in a comment for a different file, please remove the primary constructor syntax (and preserve the comments). Thanks!